Model/Payment/Api/Client.php

```diff
@@ -292,6 +293,11 @@ protected function call($url, array $data, $method = \Zend_Http_Client::POST)
                 $isSplitPayDecline = strpos($result, "There's an issue with authorizing the credit card portion") !== false;
 
                 if ($isSplitPayDecline) {
+
+                    if($this->helper->cancelSplitPaymentDeclined()){
```

Comment:
this name (cancelSplitPaymentDeclined) sort of sounds like it's an action I think, instead of just checking a boolean value, could you change the name to like isSplitPayAutoDecline or something similar?

Reply:
Ok will do